circle.yml

```diff
@@ -41,24 +41,25 @@ test:
     # See the circleCI documentation regarding parallelism
     # to understand how multiple containers can be used to
     # run subsets of tests in parallel.
-    - ./scripts/all-tests.sh:
-        timeout: 900  # if a command runs this many seconds without output, kill it
-        parallel: true
+    - echo All tests are disabled now.  # TODO: Enable tests when we're ready.
```

Comment:
Yes it did @halawa. Check the build. CircleCI got confused because of the colon.